unity/kuka-ar-unity/Assets/Project/Scripts/Connectivity/Models/Wrappers/IKRLWrapper.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Project.Scripts.Connectivity.Models.Wrappers
+{
+    public interface IKRLWrapper<in T>
```

Comment:
Is there any need for T type? If i see correctly every instance of IKRLWrapper has KRLValue type nonetheless

Reply:
Currently - no.
There was an attempt to abstract it even further, but for now it's not needed. Will come back to it later - assuming sufficient amount of time.